yivi-css/src/components/yivi-pairing-form.scss

```diff
@@ -100,6 +100,7 @@
 
   .yivi-web-pairing-loading-animation {
     @extend .yivi-web-loading-animation;
+    visibility: hidden;
```

Comment:
Do things like this keep working in this setup?
https://github.com/privacybydesign/yivi-frontend-packages/pull/7/files#diff-741dbe9fb3cad8a47f5c5c5ab9e27c9945c60f70ad85d75d043b053f3ac430e6L254

Reply:
Yes, when running the yivi-web example (with yivi-dummy and 'pairing' option enabled) I did see the loading animation go by. Since the CSS is loaded on start, and the DOM manipulation happens afterwards, the visibility setting is overwritten and thereby set to visible.